daily-challenges/2024-09/14/tung1101.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { test, expect, Browser, chromium, BrowserContext, Page } from '@playwright/test';
```

Comment:
Cái nào dùng thì import, ko thì remove đi nha em.
```suggestion
import { test, expect } from '@playwright/test';
```